### PR TITLE
New Packages: tolua, unoconv, gloobus-preview - Updated: toluapp to use lua5.1 (last known stable combo)

### PR DESCRIPTION
**Pull request by antonio-malcolm**
Added tolua, updated toluapp and Conky, to support lua-cairo bindings.
Added gloobus-preview, with unconv (used by gloobus-preview to render LibreOffice-supported documents when LibreOffice is present on the system, else, defaults to icons for .doc, .odt, and other popular office doc formats).

### CODE REVIEW

**Comment by pullmoll** [comment]
You probably did a `git pull` without `--rebase` and thus your pr is mashed with unrelated changes.
You should cherry pick your changes with `git rebase -i ...` and squash the commits.

It seems `toluapp` should install a `/usr/lib/pkgconfig/tolua++.pc` file because dependencies expect it to be there?

Please make sure there is a newline at the end of the last line in your templates or you'll confuse git. Maybe your editor removes the final newline without asking. You can see it in github's formatted output of the template files displaying a red icon with tag `No newline at end of file`.

**Comment by antonio-malcolm** [comment]
Ok, thanks, will be mindful of that.


~Antonio

**Comment by antonio-malcolm** [comment]
Just as a matter of mention- toluapp is working correctly on my system.
I've been running Conky with it since the local build.


~Antonio

On 12/02/2015 07:03 AM, Jürgen Buchmüller wrote:
>
> You probably did a |git pull| without |--rebase| and thus your pr is 
> mashed with unrelated changes.
> You should cherry pick your changes with |git rebase -i ...| and 
> squash the commits.
>
> It seems |toluapp| should install a |/usr/lib/pkgconfig/tolua++.pc| 
> file because dependencies expect it to be there?
>
> Please make sure there is a newline at the end of the last line in 
> your templates or you'll confuse git. Maybe your editor removes the 
> final newline without asking. You can see it in github's formatted 
> output of the template files displaying a red icon with tag |No 
> newline at end of file|.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/voidlinux/void-packages/pull/3100#issuecomment-161325404>.
>

-- 
/"Creative ability is best displayed with the most basic tools"/

**Comment by xtraeme** [comment]
This PR is full of unwanted and unnecessary commits... please clean them up.

**Comment by antonio-malcolm** [comment]
Done!
BTW, toluapp is building and running without issue, from my local void-packages.
Thus, I'm not certain why it is failing the travis test. I've placed the template file for toluapp, as well as tolua (a dependency), on pastebin. I would greatly appreciate if some could verify or disprove my local finding. 

tolua: http://pastebin.com/7bZ745NA
toluapp: http://pastebin.com/MyWQ2Exh

**Comment by antonio-malcolm** [comment]
I should probably mention the tolua and toluapp work is necessary for Conky's Lua-Cairo binding. Admittedly, I should have put the gloobus-preview/unoconv work into it's own request, and will certainly be mindful of that in the fututre (and, my apologies for putting it here). 

I have a spearate branch for updating Conky to work with the tolua stuffs.

Reference:
http://git.io/vmhBv
http://git.io/vRB8n
http://git.io/vY6PR

**Comment by antonio-malcolm** [comment]
Hey, is there anything else which needs to be done here? This has been hanging open for a while, now.
I'm completely fine with simply having these packages/updates in my local repo- if no one's interested in adding/using them, I'll go ahead and close this. Just figured someone else might find them useful.

File: srcpkgs/gloobus-preview/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'gloobus-preview'
+pkgname=gloobus-preview
+version=0.5.x
```
Comment (xtraeme) [inline srcpkgs/gloobus-preview/template]: `0.5.x` is not an acceptable version.

File: srcpkgs/skippy-xd/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'skippy-xd'
+pkgname=skippy-xd
+version=0.5.0
+revision=1
+build_style=gnu-makefile
+hostmakedepends="git pkg-config"
+makedepends="git libX11-devel libXext-devel libXft-devel libXcomposite-devel libXdamage-devel
```
Comment (xtraeme) [inline srcpkgs/skippy-xd/template]: `git` in `makedepends` is really useless.

File: srcpkgs/skippy-xd/template [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	freetype-devel imlib2-devel fontconfig-devel giflib-devel libjpeg-turbo-devel libpng-devel zlib-devel"
+short_desc="An Expose-like full-screen task-switcher for X11"
+maintainer="Antonio Malcolm <antonio@antoniomalcolm.com>"
+license="GPL-2"
+homepage="https://github.com/penguinpowernz/skippy-xd"
+
+do_fetch() {
+	git clone https://github.com/penguinpowernz/skippy-xd.git ${pkgname}-${version} && cd ${pkgname}-${version}
+	git checkout 33b4de5
+	git reset --hard 33b4de5
```
Comment (xtraeme) [inline srcpkgs/skippy-xd/template]: No real tag? so that it's a git snaphot, isn't it?

**Comment by xtraeme** [comment]
Please separate the packages into new PRs, so that it's easier to review them.

**Pull request closed without merging**